src/components/radios/radios.yaml

```diff
@@ -78,6 +78,22 @@ examples:
           <span class="govuk-heading-s govuk-!-margin-bottom-1">Part 3 of the Housing Act 2004</span>
           For properties that are within a geographical area defined by a local council
 
+- name: with divider
+  data:
+    idPrefix: example-divider
+    name: example
+    fieldset:
+      legend:
+        text: Have you changed your name?
+    items:
+      - value: governement-gateway
+        text: Use Government Gateway
+      - value: govuk-verify
+        text: 'Use GOV.UK Verify'
+      - divider: 'or'
```

Comment:
we could follow the same rules as with all our text params and allow for either txt or html, but surely it should only be text